src/test/resources/org/jabref/logic/importer/fetcher/collection_of_computer_science_bibliographies_empty_result.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?xml-stylesheet title="XSL_formatting" type="text/xsl" href="http://liinwww.ira.uka.de/bibliography/rss.xsl"?><rss version="2.0">
+    
+    <channel xmlns:dc="http://purl.org/dc/elements/1.1/">
+        
+        <title>CCSB: "test string which returns no results"</title>
+        <link>http://liinwww.ira.uka.de/bibliography/#search</link>
+        <description>Search results in The Collection of Computer Science Bibliographies for query: "test string which returns no results"</description>
+        <language>en</language>
```

Comment:
Interesting. In CollectionOfComputerScienceBibliographiesFetcherTest.java, there was `eng` set as language. Where did the change come from? Does CCSB provide different languages?

Reply:
This xml is for the first page returned after a search [(example)](http://liinwww.ira.uka.de/bibliography/rss?query=kopp%20decker&sort=score). But for each of the results in here, there is a link to a second page [(example)
](http://liinwww.ira.uka.de/searchbib/index?query=hmdtbhgnnqjrcfgprnkqtprfkhdrpqkq&results=bibtex&mode=dup&rss=1) which itself has a one or more BibTeX results. The BibTex results themselves can specify a language (e.g. ger for german), which is then parsed by the existing `BibTexParser`. I believe that `eng` is the default.

Reply:
The search result is interesting, because these two are the same papers ^^.

No need to invest much time in that. A `org.jabref.logic.formatter.bibtexfields.NormalizeLanguageFormatter` would be cool; but not at all required. Passing the language field as is, is perfectly fine with me. (Knowing that a new Formatter needs tests and that causes work)